.idea/misc.xml

```diff
@@ -4,18 +4,15 @@
     <option name="filePathToZoomLevelMap">
       <map>
         <entry key="..\:/jjves/Documents/SpT-KTH/Q2-MobileApplications/MoveAble/app/src/main/res/drawable/background.xml" value="0.3828125" />
-        <entry key="..\:/jjves/Documents/SpT-KTH/Q2-MobileApplications/MoveAble/app/src/main/res/drawable/background_bluetoothscan.xml" value="0.32685185185185184" />
-        <entry key="..\:/jjves/Documents/SpT-KTH/Q2-MobileApplications/MoveAble/app/src/main/res/layout/activity_bluetooth.xml" value="0.2046875" />
```

Comment:
Have you not pulled the master before working on a new branch or are those deleted on purpose?

Reply:
However according to Internet is should not give any issues so don't worry about it

Reply:
No idea, i pulled the master after the merge and defo did not delete those